hooks/610-cleanup-xml-core.chroot

```diff
@@ -1,14 +1,10 @@
-#!/bin/sh
+#!/bin/sh -ex
 #
 # Remove parts of the xml-core package - runnable bits are used by polkit
 # package postinst/prerm scripts and it is not needed after that.
 
-set -ex
```

Comment:
I prefer using `set` than in the shebang. The reason is if something calls "sh ./the-script.sh" instead of "./the-script.sh", the parameters in the shebang are lost.

Reply:
Will update the hooks to reflect this style, and thank you for making me aware of this